src/core/email_config.py

```
@@ -8,6 +8,10 @@
 from __future__ import annotations
 
 from ok import ConfigOption
+
+# WA: set empty PATH to resolve qfluentwidgets access os.environ['PATH'] issue
+if "PATH" not in os.environ:
```

Ruff (0.16.1)

[error] Undefined name `os`

(F821)